frontend/src/app/projects/models/backup-settings/job-run-overview/job-run-overview.component.ts

```diff
@@ -18,7 +25,13 @@ import { ProjectService } from 'src/app/projects/service/project.service';
   styleUrls: ['./job-run-overview.component.css'],
 })
 @UntilDestroy()
-export class JobRunOverviewComponent implements OnInit {
+export class JobRunOverviewComponent implements OnInit, AfterViewInit {
+  pageSize = 25;
+  pageSizeArray = [...Array(this.pageSize).keys()];
+
+  @ViewChildren('page', { read: ElementRef })
+  pageElements?: QueryList<ElementRef>;
+
   constructor(
```

Comment:
What is your opinion on the order of service in the constructor? I always tried to have the public services first followed by the private ones. By following this, we can easily and directly identify public and private services.

Reply:
Sounds good.